go.mod

```diff
@@ -85,7 +85,7 @@ require (
 	github.com/gravitational/license v0.0.0-20210218173955-6d8fb49b117a
 	github.com/gravitational/oxy v0.0.0-20221029012416-9fbf4c444680
 	github.com/gravitational/roundtrip v1.0.2
-	github.com/gravitational/teleport/api v0.0.0
+	github.com/gravitational/teleport/api v13.1.0
```

Comment:
Not sure if this matters since we replace the API module, but maybe we want to exclude it.